### DISCUSSION TITLE
Why was Runtime Utils replaced with Hot Reloading?

### DISCUSSION THREAD

**Discussion by ProjectHSI** [MattFiler/LegendPlugin#329] [Q&A] [answered]
It looks like Runtime Utils at one point in time got mostly gutted except for Hot Reloading, why is that?

**Comment by MattFiler** [comment]
The runtime utils project is forked from RyanJGray's devtools - perhaps that's what you mean? https://github.com/RyanJGray/AlienIsolation.DevTools

**Reply by ProjectHSI** [reply]
Your Devtools Utils fork originally was Runtime Utils, however at some point you removed nearly all of the functionality of Runtime Utils except for Hot Reloading. You have since changed the Runtime Utils option in the Launch Game settings to be Hot Reloading.

My question is why was your fork of DevTools changed into purely hot reloading? I think it would make more sense to add it in as a feature to existing feature pile, instead of removing nearly all of DevTools' features and just keeping hot reloading.

**Reply by MattFiler** [reply, marked as answer]
Yeah I'm still working with Ryan to expand the utils to provide more useful functionality for modding. The stuff that was in there originally was not high enough value for me to continue maintaining, and was kinda buggy, so I removed it to save myself the extra distractions from working on the scripting tool 👍

**Reply by ProjectHSI** [reply]
Thanks.
